docs/api-reference/geo-layers/mvt-layer.md

```diff
@@ -132,11 +132,11 @@ Remarks:
 
 ##### `onDataLoad` (Function, optional)
 
-`onDataLoad` called when a tileJSON is successfully fetched
+Called when a TileJSON is successfully fetched
```

Comment:
As far as I can tell, TileJSON is a format, talking about "a TileJSON" doesn't quite make sense.
```suggestion
Called when a TileJSON file is successfully fetched
```